Find a functionally equivalent code:android.graphics.Bitmap.setPixel:COMMENT
Method Modifier: public      
Comment:/**
 * <p>Write the specified {@link Color} into the bitmap (assuming it is
 * mutable) at the x,y coordinate. The color must be a
 * non-premultiplied ARGB value.</p>
 *
 * @param x     The x coordinate of the pixel to replace (0...width-1)
 * @param y     The y coordinate of the pixel to replace (0...height-1)
 * @param color The ARGB color to write into the bitmap
 *
 * @throws IllegalStateException if the bitmap is not mutable
 * @throws IllegalArgumentException if x, y are outside of the bitmap's
 * bounds.
 */

@@ -4,8 +4,8 @@
         throw new IllegalStateException();
     }
     checkPixelAccess(x, y);
-    nativeSetPixel(mNativeBitmap, x, y, color, mIsPremultiplied);
+    nativeSetPixel(mNativeBitmap, x, y, color);
 }

